### ISSUE TITLE
Use nimgrep for find

### ISSUE THREAD

**Issue by coffeepots** [nim-lang/Aporia#90]
The standard CTRL-F doesn't work so well with Nim's permissive naming conversion.
Incorporating nimgrep into the find would also mean Aporia could have a 'find in files' and/or 'find in directory'.

**Comment by dom96** [comment]
Ctrl+F supports style insensitive search already. I agree that "find in all files" would be a useful feature though.

**Comment by coffeepots** [comment]
You're right, I just saw the option for style insensitive! Well then, this makes this a less important feature request :)
